rust/kernel/bindings_helper.h

```diff
@@ -10,6 +10,8 @@
 #include <linux/version.h>
 #include <linux/miscdevice.h>
 #include <linux/poll.h>
+#include <linux/mm.h>
+#include <uapi/linux/android/binder.h>
```

Comment:
Sorry, late to party. But I think in the long term, maybe we don't want to put this `#include` in the `rust/kernel/bindings_helper.h`? Maybe we can support module-level `bindgen`, that is allowing a module to have its own `bindings_helper.h` and only generate when necessary.

Maybe open an issue?

Thoughts?

Reply:
+1, we discussed this the other day. There are a couple of bits of functionality that will need to be expanded to individual modules.

Reply:
Done: #181 for the documentation, #182 for the bindings.